dedupe/predicates.py

```diff
@@ -354,6 +355,14 @@ def firstTokenPredicate(field: str) -> Sequence[str]:
         return ()
 
 
+def firstTwoTokensPredicate(field: str) -> Sequence[str]:
+    first_two_tokens = two_start_words(field)
+    if first_two_tokens:
+        return (first_two_tokens.group(),)
```

Comment:
can this just be?

```
if first_tokens:
    return first_two_tokens.groups()
```

Reply:
Unfortunately that's not possible as there needs to be a capturing group in the regex for the second word (with thee '?' quantifier).